<!DOCTYPE html>
<html><head><title>AssertExpectedExceptionAsync</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">6 references to AssertExpectedExceptionAsync</div><div class="rA">Azure.Storage.Files.Shares.Tests (6)</div><div class="rG" id="Azure.Storage.Files.Shares.Tests"><div class="rF"><div class="rN">DirectoryClientTests.cs (1)</div>
<a href="../DirectoryClientTests.cs.html#667"><b>667</b>await TestHelper.<i>AssertExpectedExceptionAsync</i>&lt;ArgumentOutOfRangeException&gt;(</a>
</div>
<div class="rF"><div class="rN">FileClientTests.cs (5)</div>
<a href="../FileClientTests.cs.html#1056"><b>1056</b>await TestHelper.<i>AssertExpectedExceptionAsync</i>&lt;ArgumentOutOfRangeException&gt;(</a>
<a href="../FileClientTests.cs.html#3529"><b>3529</b>await TestHelper.<i>AssertExpectedExceptionAsync</i>&lt;ArgumentNullException&gt;(</a>
<a href="../FileClientTests.cs.html#3533"><b>3533</b>await TestHelper.<i>AssertExpectedExceptionAsync</i>&lt;ArgumentOutOfRangeException&gt;(</a>
<a href="../FileClientTests.cs.html#3537"><b>3537</b>await TestHelper.<i>AssertExpectedExceptionAsync</i>&lt;ArgumentOutOfRangeException&gt;(</a>
<a href="../FileClientTests.cs.html#3541"><b>3541</b>await TestHelper.<i>AssertExpectedExceptionAsync</i>&lt;ArgumentOutOfRangeException&gt;(</a>
</div>
</div>
</body></html>